src/main/java/ca/corefacility/bioinformatics/irida/repositories/specification/ProjectSampleSpecification.java

```diff
@@ -53,8 +53,9 @@
 			}
 			// Check for the table search.
 			// This can be expanded in future to search any attribute on the sample (e.g. description)
+			// Note: The .replace("_", "\\_") on the search term is to escape underscores
```

Comment:
This is a terrible comment.  I meant say why we are doing it (i.e. having issue with underscores and dashes being interpreted the same), and the link to the post you found the reason.

Reply:
Thanks!